.travis.yml

```diff
@@ -1,89 +1,94 @@
-            UID=0
-        - python: "2.7"
-          env:
-            TOXENV=assetlint
-            INSTALL_NPM_TOOLS=1
-        - python: "3.6"
-          env:
-            TOXENV=pytest3
-            CREATE_DB=kuma
-            INSTALL_ELASTICSEARCH=1
-            INSTALL_NPM_TOOLS=1
-    allow_failures:
-        - python: "3.6"
-          env:
-            TOXENV=pytest3
-            CREATE_DB=kuma
-            INSTALL_ELASTICSEARCH=1
-            INSTALL_NPM_TOOLS=1
+# matrix:
+#     include:
+#         - python: "2.7"
+#           env:
+#             TOXENV=pytest
+#             CREATE_DB=kuma
+#             INSTALL_ELASTICSEARCH=1
+#             INSTALL_NPM_TOOLS=1
+#         - python: "2.7"
+#           env:
+#             TOXENV=flake8
+#         - python: "2.7"
+#           env:
+#             TOXENV=docs
+#         - python: "2.7"
+#           env:
+#             TOXENV=locales
+#             CREATE_DB=kuma
+#             INSTALL_NPM_TOOLS=1
+#         - python: "2.7"
+#           env:
+#             TOXENV=docker
+#             INSTALL_DOCKER_COMPOSE=1
+#             UID=0
+#         - python: "2.7"
+#           env:
+#             TOXENV=assetlint
+#             INSTALL_NPM_TOOLS=1
+#         - python: "3.6"
+#           env:
+#             TOXENV=pytest3
+#             CREATE_DB=kuma
+#             INSTALL_ELASTICSEARCH=1
+#             INSTALL_NPM_TOOLS=1
+#     allow_failures:
+#         - python: "3.6"
+#           env:
+#             TOXENV=pytest3
+#             CREATE_DB=kuma
+#             INSTALL_ELASTICSEARCH=1
+#             INSTALL_NPM_TOOLS=1
 install:
-    - nvm install 10
-    - nvm use 10
-    - scripts/travis-install
-    - pip install -U pip
-    - pip install -r requirements/travis.txt
-    # Wait for ElasticSearch to be ready
-    - if [[ "$INSTALL_ELASTICSEARCH" == "1" ]]; then wget -q --waitretry=1 --retry-connrefused -T 10 -O - http://127.0.0.1:9200; fi;
+#     - nvm install 10
+#     - nvm use 10
+    - INSTALL_DOCKER_COMPOSE=1 scripts/travis-install
+#     - pip install -U pip
+#     - pip install -r requirements/travis.txt
+#     # Wait for ElasticSearch to be ready
+#     - if [[ "$INSTALL_ELASTICSEARCH" == "1" ]]; then wget -q --waitretry=1 --retry-connrefused -T 10 -O - http://127.0.0.1:9200; fi;
 script:
-    - tox -v
-after_failure:
-    - dmesg | tail
-notifications:
-    irc:
-        channels:
-            - "irc.mozilla.org#mdndev"
-        on_success: always
-        on_failure: always
-        use_notice: true
+      - docker-compose --version
+      - docker-compose pull
+      - docker-compose build
+      - docker-compose up
+      - docker-compose exec web npm run webpack
+      - docker-compose exec web manage.py compilejsi18n
+      - docker-compose exec web manage.py compilejsi18n -d react
+      - docker-compose exec web manage.py collectstatic --noinput
+      - docker-compose exec web pytest kuma/search -x
+      - docker-compose exec web flake8 kuma docs tests
```

Comment:
This run of `flake8` will need the following config that used to be provided by `tox.ini`:
```
exclude = **/migrations/**,.tox,*.egg,vendor
# E501 - line too long (82 > 79 characters)
# E731 - do not assign a lambda expression, use a def
# F405 - name may be undefined, or defined from star imports: module
# W504 - line break after binary operator
#        conflicts with W503 (line break before binary operator)
ignore = E501,E731,F405,W504
# flake8-import-order settings
import-order-style=edited
application-import-names=kuma,pages,utils
```

One option is that we could create a new `.flake8` file that has this configuration.